### PR TITLE
The block attribute for find_or_x_by methods should be nilable.

### LINKED ISSUE #271: find_or_create_by! shouldn't require a block parameter
I use find_or_create_by in a few places in my app, and the new signature for it causes type errors that are incorrect.

```
db/seeds/admin.rb:39: find_or_create_by! requires a block parameter, but no block was passed https://srb.help/7021
    39 |  GamePurchase.find_or_create_by!(
    40 |    game: game,
    41 |    user: admin,
    42 |    rating: rand(0..100),
    43 |    completion_status: rand(0..5),
       |...
    46 |    comments: Faker::Lorem.sentence,
    47 |    hours_played: rand(0..100),
    48 |    platforms: platforms,
    49 |    stores: stores
    50 |  )
    sorbet/rails-rbi/active_record_base.rbi:37: defined here
    37 |  def self.find_or_create_by!(attributes, &block); end
```

The block parameter for find_or_create_by! should be nilable. https://github.com/chanzuckerberg/sorbet-rails/blob/f8dbc6b03030d016160c4156f719f60d4f340f62/lib/bundled_rbi/active_record_base.rbi#L24

**Versions:**
- Ruby: 2.6.5 
- Rails: 6.0.0.1
- Sorbet: 0.5.5365
- Sorbet-Rails: git master

### PR DESCRIPTION
**Pull request by connorshea**
This applies to all three of these methods. Fixes #271.

e.g. this from my Rails app:
```ruby
GamePurchase.find_or_create_by!(
  game: game,
  user: user,
  rating: rand(0..100),
  completion_status: rand(0..5),
  start_date: Faker::Date.between(from: 1.month.ago, to: 1.day.ago),
  completion_date: Faker::Date.between(from: 1.month.ago, to: 1.day.ago),
  comments: Faker::Lorem.sentence,
  platforms: platforms,
  stores: stores
)
```

### CODE REVIEW

**Comment by mahmoudian1** [comment]
# [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=h1) Report
> Merging [#272](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=desc) into [master](https://codecov.io/gh/chanzuckerberg/sorbet-rails/commit/f8dbc6b03030d016160c4156f719f60d4f340f62&el=desc) will **decrease** coverage by `79.78%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree)

```diff
@@             Coverage Diff             @@
##           master     #272       +/-   ##
===========================================
- Coverage   96.73%   16.95%   -79.79%     
===========================================
  Files         140       34      -106     
  Lines        2728      696     -2032     
===========================================
- Hits         2639      118     -2521     
- Misses         89      578      +489     
```


| [Impacted Files](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [spec/sorbet\_utils\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zb3JiZXRfdXRpbHNfc3BlYy5yYg==) | `1.36% <0.00%> (-98.64%)` | :arrow_down: |
| [spec/custom\_params\_methods\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9jdXN0b21fcGFyYW1zX21ldGhvZHNfc3BlYy5yYg==) | `1.49% <0.00%> (-98.51%)` | :arrow_down: |
| [spec/custom\_finder\_methods\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9jdXN0b21fZmluZGVyX21ldGhvZHNfc3BlYy5yYg==) | `2.32% <0.00%> (-97.68%)` | :arrow_down: |
| [spec/typed\_enum\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy90eXBlZF9lbnVtX3NwZWMucmI=) | `2.94% <0.00%> (-97.06%)` | :arrow_down: |
| [spec/model\_rbi\_formatter\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9tb2RlbF9yYmlfZm9ybWF0dGVyX3NwZWMucmI=) | `3.57% <0.00%> (-96.43%)` | :arrow_down: |
| [spec/integer\_string\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9pbnRlZ2VyX3N0cmluZ19zcGVjLnJi) | `3.70% <0.00%> (-96.30%)` | :arrow_down: |
| [spec/routes\_rbi\_formatter\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9yb3V0ZXNfcmJpX2Zvcm1hdHRlcl9zcGVjLnJi) | `5.55% <0.00%> (-94.45%)` | :arrow_down: |
| [spec/rake\_rails\_rbi\_models\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9yYWtlX3JhaWxzX3JiaV9tb2RlbHNfc3BlYy5yYg==) | `5.55% <0.00%> (-94.45%)` | :arrow_down: |
| [spec/model\_plugins\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9tb2RlbF9wbHVnaW5zX3NwZWMucmI=) | `5.88% <0.00%> (-94.12%)` | :arrow_down: |
| [spec/helper\_rbi\_formatter\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9oZWxwZXJfcmJpX2Zvcm1hdHRlcl9zcGVjLnJi) | `7.14% <0.00%> (-92.86%)` | :arrow_down: |
| ... and [129 more](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree-more) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=footer). Last update [f8dbc6b...9b23c8d](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by mahmoudian1** [comment]
# [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=h1) Report
> Merging [#272](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=desc) into [master](https://codecov.io/gh/chanzuckerberg/sorbet-rails/commit/f8dbc6b03030d016160c4156f719f60d4f340f62&el=desc) will **decrease** coverage by `81.29%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree)

```diff
@@             Coverage Diff             @@
##           master     #272       +/-   ##
===========================================
- Coverage   96.73%   15.44%   -81.30%     
===========================================
  Files         140       31      -109     
  Lines        2728      680     -2048     
===========================================
- Hits         2639      105     -2534     
- Misses         89      575      +486     
```


| [Impacted Files](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [spec/sorbet\_utils\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zb3JiZXRfdXRpbHNfc3BlYy5yYg==) | `1.36% <0.00%> (-98.64%)` | :arrow_down: |
| [spec/custom\_params\_methods\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9jdXN0b21fcGFyYW1zX21ldGhvZHNfc3BlYy5yYg==) | `1.49% <0.00%> (-98.51%)` | :arrow_down: |
| [spec/custom\_finder\_methods\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9jdXN0b21fZmluZGVyX21ldGhvZHNfc3BlYy5yYg==) | `2.32% <0.00%> (-97.68%)` | :arrow_down: |
| [spec/typed\_enum\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy90eXBlZF9lbnVtX3NwZWMucmI=) | `2.94% <0.00%> (-97.06%)` | :arrow_down: |
| [spec/model\_rbi\_formatter\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9tb2RlbF9yYmlfZm9ybWF0dGVyX3NwZWMucmI=) | `3.57% <0.00%> (-96.43%)` | :arrow_down: |
| [spec/integer\_string\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9pbnRlZ2VyX3N0cmluZ19zcGVjLnJi) | `3.70% <0.00%> (-96.30%)` | :arrow_down: |
| [spec/routes\_rbi\_formatter\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9yb3V0ZXNfcmJpX2Zvcm1hdHRlcl9zcGVjLnJi) | `5.55% <0.00%> (-94.45%)` | :arrow_down: |
| [spec/rake\_rails\_rbi\_models\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9yYWtlX3JhaWxzX3JiaV9tb2RlbHNfc3BlYy5yYg==) | `5.55% <0.00%> (-94.45%)` | :arrow_down: |
| [spec/model\_plugins\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9tb2RlbF9wbHVnaW5zX3NwZWMucmI=) | `5.88% <0.00%> (-94.12%)` | :arrow_down: |
| [spec/helper\_rbi\_formatter\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9oZWxwZXJfcmJpX2Zvcm1hdHRlcl9zcGVjLnJi) | `7.14% <0.00%> (-92.86%)` | :arrow_down: |
| ... and [129 more](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree-more) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=footer). Last update [f8dbc6b...9b23c8d](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by mahmoudian1** [comment]
# [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=h1) Report
> Merging [#272](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=desc) into [master](https://codecov.io/gh/chanzuckerberg/sorbet-rails/commit/f8dbc6b03030d016160c4156f719f60d4f340f62&el=desc) will **decrease** coverage by `1.15%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #272      +/-   ##
==========================================
- Coverage   96.73%   95.57%   -1.16%     
==========================================
  Files         140      124      -16     
  Lines        2728     2239     -489     
==========================================
- Hits         2639     2140     -499     
- Misses         89       99      +10     
```


| [Impacted Files](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [spec/support/v6.0/config/environment.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y2LjAvY29uZmlnL2Vudmlyb25tZW50LnJi) | `50.00% <0.00%> (-50.00%)` | :arrow_down: |
| [...rbet-rails/model\_plugins/active\_storage\_methods.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-bGliL3NvcmJldC1yYWlscy9tb2RlbF9wbHVnaW5zL2FjdGl2ZV9zdG9yYWdlX21ldGhvZHMucmI=) | `50.00% <0.00%> (-50.00%)` | :arrow_down: |
| [spec/support/v6.0/config/application.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y2LjAvY29uZmlnL2FwcGxpY2F0aW9uLnJi) | `75.00% <0.00%> (-25.00%)` | :arrow_down: |
| [lib/sorbet-rails/utils.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-bGliL3NvcmJldC1yYWlscy91dGlscy5yYg==) | `90.90% <0.00%> (-9.10%)` | :arrow_down: |
| [spec/support/v5.0/config/boot.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjAvY29uZmlnL2Jvb3QucmI=) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v6.0/config/boot.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y2LjAvY29uZmlnL2Jvb3QucmI=) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v5.0/config/routes.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjAvY29uZmlnL3JvdXRlcy5yYg==) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v5.2/config/routes.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjIvY29uZmlnL3JvdXRlcy5yYg==) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v5.0/sorbet\_test\_cases.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjAvc29yYmV0X3Rlc3RfY2FzZXMucmI=) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v5.1/sorbet\_test\_cases.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjEvc29yYmV0X3Rlc3RfY2FzZXMucmI=) | `100.00% <0.00%> (ø)` | :arrow_up: |
| ... and [44 more](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree-more) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=footer). Last update [f8dbc6b...9b23c8d](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: lib/sorbet-rails/model_plugins/active_record_finder_methods.rb [hunk 1]
```diff
@@ -60,7 +63,10 @@ def create_finder_methods_for(class_rbi, class_method:, include_methods_that_ret
         "find_or_create_by",
         parameters: [
           Parameter.new("attributes", type: "T.untyped") ,
-          Parameter.new("&block", type: "T.proc.params(object: #{self.model_class_name}).void"),
+          Parameter.new(
+            "&block",
+            type: "T.nilable(T.proc.params(object: #{self.model_class_name})).void"
```
Comment (connorshea) [inline lib/sorbet-rails/model_plugins/active_record_finder_methods.rb]: I think you typo'd on some of these, they're not generating valid RBIs:
```suggestion
            type: "T.nilable(T.proc.params(object: #{self.model_class_name}).void)"
```

File: lib/sorbet-rails/model_plugins/active_record_finder_methods.rb [hunk 2]
```diff
@@ -51,7 +51,10 @@ def create_finder_methods_for(class_rbi, class_method:, include_methods_that_ret
         "find_or_initialize_by",
         parameters: [
           Parameter.new("attributes", type: "T.untyped") ,
-          Parameter.new("&block", type: "T.proc.params(object: #{self.model_class_name}).void"),
+          Parameter.new(
+            "&block",
+            type: "T.nilable(T.proc.params(object: #{self.model_class_name})).void",
```
Comment (connorshea) [inline lib/sorbet-rails/model_plugins/active_record_finder_methods.rb]:
```suggestion
            type: "T.nilable(T.proc.params(object: #{self.model_class_name}).void)",
```

File: lib/sorbet-rails/model_plugins/active_record_finder_methods.rb [hunk 3]
```diff
@@ -69,7 +75,10 @@ def create_finder_methods_for(class_rbi, class_method:, include_methods_that_ret
         "find_or_create_by!",
         parameters: [
           Parameter.new("attributes", type: "T.untyped") ,
-          Parameter.new("&block", type: "T.proc.params(object: #{self.model_class_name}).void"),
+          Parameter.new(
+            "&block",
+            type: "T.nilable(T.proc.params(object: #{self.model_class_name})).void",
```
Comment (connorshea) [inline lib/sorbet-rails/model_plugins/active_record_finder_methods.rb]:
```suggestion
            type: "T.nilable(T.proc.params(object: #{self.model_class_name}).void)",
```

**Comment by connorshea** [comment]
@manhhung741 make sure to regenerate the expectations before merging this :)

**Comment by mahmoudian1** [comment]
# [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=h1) Report
> Merging [#272](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=desc) into [master](https://codecov.io/gh/chanzuckerberg/sorbet-rails/commit/f8dbc6b03030d016160c4156f719f60d4f340f62&el=desc) will **decrease** coverage by `1.09%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master     #272      +/-   ##
==========================================
- Coverage   96.45%   95.35%   -1.10%     
==========================================
  Files         140      124      -16     
  Lines        2739     2241     -498     
==========================================
- Hits         2642     2137     -505     
- Misses         97      104       +7     
```


| [Impacted Files](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [spec/support/v6.0/config/environment.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y2LjAvY29uZmlnL2Vudmlyb25tZW50LnJi) | `50.00% <0.00%> (-50.00%)` | :arrow_down: |
| [...rbet-rails/model\_plugins/active\_storage\_methods.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-bGliL3NvcmJldC1yYWlscy9tb2RlbF9wbHVnaW5zL2FjdGl2ZV9zdG9yYWdlX21ldGhvZHMucmI=) | `50.00% <0.00%> (-50.00%)` | :arrow_down: |
| [spec/support/v6.0/config/application.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y2LjAvY29uZmlnL2FwcGxpY2F0aW9uLnJi) | `75.00% <0.00%> (-25.00%)` | :arrow_down: |
| [spec/rake\_rails\_rbi\_models\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9yYWtlX3JhaWxzX3JiaV9tb2RlbHNfc3BlYy5yYg==) | `77.77% <0.00%> (-22.23%)` | :arrow_down: |
| [lib/sorbet-rails/utils.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-bGliL3NvcmJldC1yYWlscy91dGlscy5yYg==) | `90.90% <0.00%> (-9.10%)` | :arrow_down: |
| [spec/sorbet\_spec.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zb3JiZXRfc3BlYy5yYg==) | `81.81% <0.00%> (-3.04%)` | :arrow_down: |
| [spec/support/v5.0/config/boot.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjAvY29uZmlnL2Jvb3QucmI=) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v6.0/config/boot.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y2LjAvY29uZmlnL2Jvb3QucmI=) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v5.0/config/routes.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjAvY29uZmlnL3JvdXRlcy5yYg==) | `100.00% <0.00%> (ø)` | :arrow_up: |
| [spec/support/v5.2/config/routes.rb](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree#diff-c3BlYy9zdXBwb3J0L3Y1LjIvY29uZmlnL3JvdXRlcy5yYg==) | `100.00% <0.00%> (ø)` | :arrow_up: |
| ... and [48 more](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272/diff?src=pr&el=tree-more) | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=footer). Last update [f8dbc6b...d533293](https://codecov.io/gh/chanzuckerberg/sorbet-rails/pull/272?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by manhhung741** [comment]
Thanks for catching 🙌 
In order to avoid unfortunate mistakes like this, I intend to test this against my repo before merging/releasing.

**Comment by manhhung741** [comment]
(I wonder why sorbet tc still passed. Shouldn't it flag the typo?)

**Pull request merged**